FWCore/Services/plugins/CondorStatusUpdater.cc

```diff
@@ -338,6 +339,24 @@ CondorStatusService::updateImpl(time_t sinceLastUpdate)
     }
 
     // Update storage account information
+    auto &stats = StorageAccount::summary();
+    updateStorageImpl(stats);
+    {
+      // Given there is some cost to reporting empty numbers, only push forward the secondary source info
+      // if one file was actually opened via such a source.
+      auto label = StorageAccount::setLabel("secondarysource");
+      const auto token = StorageAccount::tokenForStorageClassName("tstoragefile");
+      auto &operations = stats[token.value()];
```

Comment:
Changing `StorageAccount::summary` to return a non-const object just so you can call `operator[]` does not seem reasonable to me. You could just call `find` on the const object.